rc/core/comment.kak

```diff
@@ -3,41 +3,50 @@ decl -docstring "characters inserted at the beginning of a commented line" \
     str comment_line '#'
 
 ## Block comments
-decl -docstring "colon separated tuple containing the characters inserted before/after a commented line" \
-    str-list comment_block
+decl -docstring "characters inserted before a commented block" \
+    str comment_block_begin
+decl -docstring "characters inserted after a commented block" \
+    str comment_block_end
 
 ## Default comments for all languages
 hook global BufSetOption filetype=asciidoc %{
-    set buffer comment_block '///:///'
+    set buffer comment_block_begin '///'
+    set buffer comment_block_end '///'
```

Comment:
I originally stored the separators in a single command to keep things simple, why do we need two now? Parsing it is trivial with shell expansions, as long as no language needs a colon `:` as a commenting character.